ONSdigital/response-operations-ui
# 603 my conversations

# Motivation and Context
The internal users want the ability to have 'my messages' tab. This is conversations where they were the last internal person to reply or if they sent a first message. 
This change adds that tab. 
Note we now have three 'tabs' my messages, open and close. Messages that appear in my messages also appear in open. So that any internal user can still access any conversation.

# What has changed
Added extra tab. my conversations is now an optional parameter that can be sent to secure message service 

# How to test?
Acceptance tests cover most of it. Bu the easiest way is to send a message from respondent. Should only appear in open messages. Reply as internal and it should also be visible in my messages. Change user and validate that it does not show in my messages for new internal user .But can be seen in open. Send several messages from respondent (more than 1 page worth) view in my messages and open. Validate that back goes back to the correct tab and page

# Links
 https://trello.com/c/djNU7lM5
